### ISSUE TITLE
Adding Favorite contacts after installing AIO Tweaks 2.8.0 froze the MZD

### ISSUE THREAD

**Issue by Tristan-cx5** [Trevelopment/MZD-AIO#16]
Hi Trez,
my setup: FW 59.00.545A EU (installed by dealer to my new CMU build in 2018-04), CX-5 2016, AIO 2.8.0, serial access done by myself, installed speedcam patch from diginix and ID7 Tweak before.
Today I installed the Tweaks via AIO 2.8.0 you can see in the attachment. Afterwards I wanted to add some contacts to the favourites list of the MZD (from my Samsung Galaxy S7 with Android 8 connected via Bluetooth). But several times the MZD froze after adding 1-3 contacts - at different buttons. The button stayed orange. Only a reboot via Nav-Mute was possible (did it 3-4 times). Then I uninstalled all tweaks via "uninstall all" compilation in AIO. Afterwards there was a constant boot logo (loop?) with the spinner visibile. Even after acc&power off & rebooting. So I decided to use the USB stick with "uninstall all" again - and this worked. Afterwards a normal reboot was possible. All tweaks were removed. Adding several favorite contacts and addresses worked without freeze this time. So probably one of the tweaks I installed lead to the freeze at adding contacts. Do you have an idea which it could be? I will try installing the tweaks you can see in the other attached photo.
Cheers
Tristan-cx5

**Comment by Tristan-cx5** [comment]
And here the attachments:
![tristan-cx5_20180731_aio280_freeze_adding_favorites](https://user-images.githubusercontent.com/34833631/43477803-6776162c-94fc-11e8-944e-90ed996875ab.jpg)
![tristan-cx5_20180731_aio280_next try](https://user-images.githubusercontent.com/34833631/43477804-6797ebb2-94fc-11e8-8b3c-0aee366ed921.jpg)

**Comment by Tristan-cx5** [comment]
My guess does not seem to be the cause. Just right now I installed the tweaks shown above (next time). After adding 3 to the existing 5 favorite contacts (MZD) during renaming one of them the MZD froze again with the orange color on the button - see photo:
![20180731_next try_also_freeze](https://user-images.githubusercontent.com/34833631/43489641-f52fe9e0-951d-11e8-8a5c-373d7e58694c.jpg)

**Comment by Trevelopment** [comment]
I think it maybe has something to do with the "Remove Message Reply" Tweak maybe?  I feel like all the other tweaks would give you an issue right after booting up if they were going to give you problems at all.  BTW did you do the factory reset after installing "Remove Message Reply" tweak to apply it?  

But you're saying it stops happening when you uninstall all the apps? so can you try I guess the only others it could be are "Pause On Mute", or "No More Disclaimer & Audio Source Order" because those are on the same file so you can just think of them together as one.  The rest are pretty much just .png images & the 2 apps.

**Comment by Tristan-cx5** [comment]
No, I did not do the factory reset after installing "Remove Message Reply". As I already had set my favorites (radio/contacts/locations) & did not want to loose them. This tweak is not so important, so that I have uninstalled it and will not install it again.

This morning I uninstalled the tweaks you mentioned one by one. After each uninstall I added favorites and got the freeze as before. Afterwards I uninstalled all of your mentioned tweaks at once - just to be sure - afterwards I also got a freeze while ading favorite contacts.

I removed: Remove Message Reply, Pause on Mute, No more Disclaimer, Audio Source Order.

Probably I will test the remaining tweaks as well.

The bug/freeze does not occur when no tweaks are installed.

But to be fair - it is just a minor bug, as you are not modifing your favorite contacts every day.

**Comment by Tristan-cx5** [comment]
Yesterday I tested again for more than an hour. The result: I might have forgotten to uninstall the "No More beeps"-Tweak in the first tests. Because in the tests yesterday the freeze-problem only occured, with the installed "No More beeps"-Tweak. I had installed the tweaks shown in the screenshot below + the "No More beeps"-Tweak. For testing I added, renamed and relocated always the same 5 contacts/favorites. Afterwards I deleted them - if the freeze did not occur before. Within the test cycle the freeze occured at different buttons. With all my other tweaks installed, there was no freeze. So maybe you or the developer of the "No More beeps"-Tweak can have a look in the compatibility with FW 59.00.545?

At the moment I have the following tweaks installed and ran 2 test cycles without any freeze:
![aio280_current_tweaks_tristan](https://user-images.githubusercontent.com/34833631/43627472-da6f3fc2-96f6-11e8-89ff-e9c0bc153959.JPG)
